lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -226,14 +227,17 @@ export abstract class BaseManagedIdentitySource {
 
     public getManagedIdentityUserAssignedIdQueryParameterKey(
         managedIdentityIdType: ManagedIdentityIdType,
-        imds?: boolean
+        imds?: boolean,
+        api2017?: boolean
     ): string {
         switch (managedIdentityIdType) {
             case ManagedIdentityIdType.USER_ASSIGNED_CLIENT_ID:
                 this.logger.info(
                     "[Managed Identity] Adding user assigned client id to the request."
                 );
-                return ManagedIdentityUserAssignedIdQueryParameterNames.MANAGED_IDENTITY_CLIENT_ID;
+                return api2017
+                    ? ManagedIdentityUserAssignedIdQueryParameterNames.MANAGED_IDENTITY_CLIENT_ID_2017
+                    : ManagedIdentityUserAssignedIdQueryParameterNames.MANAGED_IDENTITY_CLIENT_ID_2019;
```

Comment:
same comment as above, you just have this as `MANAGED_IDENTITY_CLIENT_ID`

Reply:
done